src/components/ContactsList/_modals/VerifyModal/VerifyModal.tsx

```diff
@@ -0,0 +1,74 @@
+import { Modal } from '@/components/Modal'
+import { FC, PropsWithChildren } from 'react'
+import { Contact, ContactAwaitingAcceptance, ContactConfirmed, ContactNotConfirmed } from '@/api/types/contact'
+import { useYourTOTP, YourTOTP } from './YourTOTP'
+import { colors } from '@/styles'
+import { PeerTOTP, usePeerTOTP } from './PeerTOTP'
+import styled from '@emotion/styled'
+
+type VerifyModalProps = {
+  peer: Contact
+  onConfirmed: () => void
+  onClose: () => void
+}
+
+export const VerifyModal: FC<PropsWithChildren<VerifyModalProps>> = ({ children, peer, onConfirmed, onClose }) => {
+  const yourTOTP = useYourTOTP(peer.paymail)
```

Comment:
Here I have a point not to destructing the `peer` object. 

Because in this matter, it's easy to mixed up data for `peer` (the one on the other side, who you make a trancation) and data for you (the currently logged user).

Reply:
ok I will use destructing